includes/admin/payments/class-refund-items-table.php

```diff
@@ -191,59 +295,88 @@ private function format_currency( $order_item, $column_name ) {
-		if ( 'refunded' !== $order_item->status ) {
-			$order_item_title = '<label for="edd-order-item-' . esc_attr( $order_item->id ) . '" class="edd-order-item__label">' . $order_item->get_order_item_name() . '</label>';
+	private function get_currency_symbol( $order_id ) {
+		static $symbol = null;
+
+		if ( is_null( $symbol ) ) {
+			$order = edd_get_order( $order_id );
+
+			if ( $order ) {
+				$symbol = edd_currency_symbol( $order->currency );
+			}
 		}
 
-		return $order_item_title;
+		return $symbol;
 	}
 
 	/**
 	 * Render the checkbox column
 	 *
 	 * @since 3.0
 	 *
-	 * @param \EDD\Orders\Order_Item $order_item Order Item object.
+	 * @param Order_Item|Order_Adjustment $item Order Item or Order Adjustment object.
 	 *
-	 * @return string Displays a checkbox
+	 * @return string
 	 */
-	public function column_cb( $order_item ) {
-		if ( 'refunded' !== $order_item->status ) {
+	public function column_cb( $item ) {
+		$item_id            = $this->get_item_unique_id( $item );
+		$object_type        = $this->get_object_type( $item );
+		$item_quantity      = $item instanceof Order_Item ? $item->quantity : 1;
+		$refundable_amounts = $item->get_refundable_amounts();
+		$total_remaining    = array_key_exists( 'total', $refundable_amounts ) ? floatval( $refundable_amounts['total'] ) : 0.00;
+
+		if ( 'refunded' !== $item->status && $total_remaining > 0 ) {
+			$quantity_html = '';
+			if ( ! edd_item_quantities_enabled() ) {
+				$quantity_html = '<input type="hidden" id="edd-order-item-quantity-' . esc_attr( $item_id ) . '" class="edd-order-item-refund-quantity edd-order-item-refund-input" name="refund_' . esc_attr( $object_type ) . '[' . esc_attr( $item->id ) . '][quantity]" value="' . esc_attr( $item_quantity ) . '" min="0" max="' . esc_attr( $item_quantity ) . '" />';
+			}
+
 			return sprintf(
-				'<input type="checkbox" name="%1$s[]" id="%1$s-%2$s" value="%2$s" /><label for="%1$s-%2$s" class="screen-reader-text">%3$s</label>',
+				'<input type="checkbox" name="%1$s[]" id="%1$s-%2$s" class="edd-order-item-refund-checkbox" value="%2$s" /><label for="%1$s-%2$s" class="screen-reader-text">%3$s</label>' . $quantity_html,
 				/*$1%s*/
-				'order_item',
+				'refund_' . esc_attr( $object_type ),
 				/*$2%s*/
-				esc_attr( $order_item->id ),
+				esc_attr( $item->id ),
 				/* translators: product name */
-				esc_html( sprintf( __( 'Select %s', 'easy-digital-downloads' ), $order_item->product_name ) )
+				esc_html( sprintf( __( 'Select %s', 'easy-digital-downloads' ), $this->get_item_display_name( $item ) ) )
 			);
 		}
+
+		return '';
+	}
+
+	/**
+	 * Render the Name Column
+	 *
+	 * @since 3.0
+	 *
+	 * @param Order_Item|Order_Adjustment $item Order Item object.
+	 *
+	 * @return string Data shown in the Name column
+	 */
+	public function column_name( $item ) {
+		return '<span class="row-title">' . esc_html( $this->get_item_display_name( $item ) ) . '</span>';
```

Comment:
Could this be reverted somewhat to what it is in 3.0? ([ref](https://github.com/easydigitaldownloads/easy-digital-downloads/blob/release/3.0/includes/admin/payments/class-refund-items-table.php#L365))

I think the `row-title` class is better than the `strong`, but as I'm testing/revising CSS, I'm remembering that we had intentionally made the refundable items output be a `label` element because I keep being lazy and clicking the order item name instead of the checkbox.